frontend/src/entities/configuration/configuration-history-table.tsx

```diff
@@ -36,7 +36,8 @@ export function ConfigurationHistoryTable () {
 
     useMemo(() => {
         dispatch(listConfigurations({configScope: 'global', numVersions: applicationConfig.versionId + 1}));
-    }, [applicationConfig.versionId, dispatch]);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
What is the issue here that we need to disable exhaustive-deps?

Reply:
No issue, will remove, I stole this from an issue I had someone fix while I was working on the refactor.